app.vue

```diff
@@ -67,47 +66,49 @@ const configStage = {
   height: 500,
 }
 const forceVectors = ref([])
+const vectorIndex = ref(0)
```

Comment:
If you use vector.id as the key, then you can remove this completely, I think

Reply:
Thanks for catching this. My implementation here was one of a few ideas I'd tried to make sure the draggable circle would re-render to the vector head. I tried a few things at once and it looks like this one wasn't necessary.